apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -623,17 +623,17 @@ void wm_auto_test(void *arg)
 
 	/* Set SoftAP Configuration */
 	wifi_manager_softap_config_s softap_config;
-	strncpy(softap_config.ssid, info->softap_ssid, 33);
+	strncpy(softap_config.ssid, info->softap_ssid, WIFIMGR_SSID_LEN);
 	softap_config.ssid[strlen(info->ssid)] = '\0';
-	strncpy(softap_config.passphrase, info->softap_password, 64);
+	strncpy(softap_config.passphrase, info->softap_password, WIFIMGR_PASSPHRASE_LEN);
 	softap_config.passphrase[strlen(info->password)] = '\0';
 	softap_config.channel = 1;
 
 	/* Set AP Configuration */
 	wifi_manager_ap_config_s ap_config;
-	strncpy(ap_config.ssid, info->ssid, 33);
+	strncpy(ap_config.ssid, info->ssid, WIFIMGR_SSID_LEN);
```

Comment:
Even though ssid element has ```WIFIMGR_SSID_LEN``` size, you copy it with same size. Is it ok not saving ```'\0'``` at end of array?
Usaually in this case, set array size as ```XXX + 1``` and copy it with ```XXX``` size.

Reply:
Hi @sunghan-chang, Basically, the length of SSID should be less than WIFIMGR_SSID_LEN. If the SSID entered by the user is shorter than WIFIMGR_SSID_LEN, there is no need to insert a null because the SSID variable in 'info' structure ends with null. In case that a user sets SSID of AP (or SoftAP) longer than WIFIMGR_SSID_LEN, WI-Fi Manager will return 'fail' due to length limit, that is, there is no Wi-Fi connection and no SoftAP mode using the SSID longer than WIFIMGR_SSID_LEN.
For safety reasons, if NULL is inserted at the end, only SSID of the cutted 31 bytes will be utilized when the user enters a long SSID. Which way could be a better example? We might use strcpy, instead of strncpy.

Reply:
@sunghan-chang Modification completed.